cumulusci/core/config/project_config.py

```diff
@@ -385,6 +398,9 @@ def _get_latest_tag_for_prefix(self, repo, prefix):
             if not release.tag_name.startswith(prefix):
```

Comment:
I wrote this alternative but it may not be better enough to be worth including. Up to you:

```
    def _get_latest_tag_for_prefix(self, repo, prefix):
        relevant_releases = (
            release
            for release in repo.releases()
            if release.tag_name.startswith(prefix)
        )
        first_release = next(relevant_releases, None)
        if first_release:
            return first_release.tag_name
        else:
            raise GithubException(
                f"No release found for {self.repo_url} with tag prefix {prefix}"
            )
```

Reply:
To be honest I think it's easier to follow with a simple loop instead of a generator expression.

Reply:
Just for the record, my thinking is that you can follow the logic just by reading the variable names:

````
    relevant_releases = (whatever)
    first_release = next(relevant_releases, None)
    if first_release:
            return first_release.tag_name
 ```

Reply:
But its more lines of code so, shrug.

Reply:
And you have to know about generators to understand that it's lazy.